core/server/src/main/java/alluxio/master/block/BlockMasterClientRestServiceHandler.java

```diff
@@ -43,10 +48,20 @@
   public static final String SERVICE_VERSION = "service_version";
   public static final String GET_BLOCK_INFO = "block_info";
   public static final String GET_CAPACITY_BYTES = "capacity_bytes";
+  public static final String GET_DISK_CAPACITY_BYTES = "disk_capacity_bytes";
```

Comment:
because I found that `capacity_bytes`, `used_bytes`, and `worker_info_list` are put here, the added endpoints are similar concepts.